docs/user-manual/modules/ROOT/pages/camel-3x-upgrade-guide-3_15.adoc

```diff
@@ -37,6 +37,10 @@ node in the JMX MBean tree.
 
 Removed deprecated operations on `CamelContextMBean` and `CamelRouteMBean`.
 
+=== camel-util
+
+Deleted the `replaceAll` method of `org.apache.camel.util.StringHelper`.
```

Comment:
Can you say that users should use the replace method from the JDK instead

Reply:
Sure, done.